manifests/service.pp

```diff
@@ -4,11 +4,16 @@
 class candlepin::service {
   assert_private()
 
+  package { ['java-11-openjdk']:
+    ensure => installed,
+  }
+
   service { 'tomcat':
```

Comment:
We need to specify candlepin > 3.2.6 now. Not sure of a good way to do that. Any ideas?

Reply:
That's probably something to document in the README.